packages/web/src/SplunkSpanAttributesProcessor.ts

```diff
@@ -43,11 +42,14 @@ export class SplunkSpanAttributesProcessor implements SpanProcessor {
 	}
 
 	onStart(span: Span): void {
+		// now that we have started the span, the session is not a shadow anymore
+		const sessionState = updateSessionStatus({ forceStore: false, shadow: false });
```

Comment:
we just need to be sure the sampler is called before this hook. I guess it should but I was unable to find it in the otel codebase yet.